pip/req/req_set.py

```diff
@@ -170,6 +170,7 @@ def _walk_req_to_install(self, handler):
                 discovered_reqs.extend(more_reqs)
 
     def locate_files(self):
+        """Remove in 7.0: used by --no-download"""
```

Comment:
would prefer not to burn in comments like this that might not happen for whatever reason.